src/DeviceInterface/AppleDeviceInterface.js

```diff
@@ -322,6 +323,10 @@ class AppleDeviceInterface extends InterfacePrototype {
             poll()
         })
     }
+
+    firePixel (pixelName) {
+        this.deviceApi.notify(new SendJSPixelCall({pixelName}))
+    }
```

Comment:
If/when all platforms support the js pixel, we can move this implementation to the InterfacePrototype, but for now I made this Apple-only.